pkg/receiver/smartagent/README.md

```diff
@@ -1,51 +1,52 @@
+and a 
+[SignalFx exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/signalfxexporter/README.md),
+as each pipeline's exporter, respectively.
+1. To replace or modify metrics, use
 [Collector processors](https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/README.md).
-1. Monitors with [dimension property and tag update
-functionality](https://dev.splunk.com/observability/docs/datamodel#Creating-or-updating-custom-properties-and-tags)
-allow an associated `dimensionClients` field that references the name of the SignalFx exporter you are using in your
-pipeline.  These monitors include `ecs-metadata`, `heroku-metadata`, `kubernetes-cluster`, `openshift-cluster`, `postgresql`,
-and `sql`.
-If you do not specify any exporters via this field, the receiver will attempt to use the associated
-pipeline.  If the next element of the pipeline isn't compatible with the dimension update behavior, and if you configured
-a single SignalFx exporter for your deployment, the exporter will be selected.  If no dimension update behavior is desired,
-you can specify the empty array `[]` to disable.
-1. Monitors with [event-sending
-functionality](https://dev.splunk.com/observability/docs/datamodel/ingest#Send-custom-events) should also be made members of
-a `logs` pipeline that utilizes a [SignalFx
-exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/signalfxexporter/README.md)
-that will make the event submission requests. It's recommended, and in the case of the Processlist monitor required,
-to use a [Resource Detection
-processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/resourcedetectionprocessor/README.md)
-to ensure that host identity and other useful information is made available as event dimensions.
-Receiver entries that should be added to logs pipelines include `kubernetes-events`, `nagios`, `processlist`, and potentially any
-`telegraf/*` monitors like `telegraf/exec`.  The `signalfx` exporter is required for sending events to SignalFx, the `splunk_hec` exporter does not support sending events. An example of this is provided below.
+1. If you have a monitor that sends [events](https://dev.splunk.com/observability/docs/datamodel/custom_events) (e.g. `kubernetes-events`,
+`nagios`, `processlist`, and some `telegraf` monitors like `telegraf/exec`), put it in a `logs` pipeline that uses a
+[SignalFx exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/signalfxexporter/README.md).
+It's recommended, and in the case of the Processlist monitor required, to put into the same pipeline a
+[Resource Detection processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/resourcedetectionprocessor/README.md),
+which will add host information and other useful dimensions to the events. An example is provided below.
```

Comment:
```suggestion
1. If you have a monitor that sends [events](https://dev.splunk.com/observability/docs/datamodel/custom_events), for example `kubernetes-events`,
`nagios`, `processlist`, and some `telegraf` monitors like `telegraf/exec`, put it in a `logs` pipeline that uses a
[SignalFx exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/signalfxexporter/README.md).
If the Processlist monitor is required, add to the same pipeline a
[Resource Detection processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/resourcedetectionprocessor/README.md),
which adds host information and other useful dimensions to the events. See the example.
```

I suggest linking to the example. Positional language ("below") isn't very useful in this case.

Reply:
Mind if we hand this off to your team?